src/picongpu/include/simulation_defines/unitless/ionizationEnergies.unitless

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2015 Marco Garten
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** \file
+ * This file contains the ionization energies and conversion to corresponding
+ * electric field strengths in atomic units.
+ */
+
+#pragma once
+
+namespace picongpu
+{
+
+    /* \todo We definitely need something nicer to transform an arbitrarily shaped
+     * vector of ionization energies into atomic units
+     */
+    /* ionization energy for ground state hydrogen in atomic units */
+    PMACC_CONST_VECTOR(float_X,1,IONIZATION_ENERGY_AU,
+        float_X(2.18e-18 / SI::ATOMIC_UNIT_ENERGY)
```

Comment:
oh I get that ... so am I
the 2.18e-18 is the one entry from `SI::IONIZATION_ENERGY` and I'd be more than happy to have a generic conversion to atomic units - I just don't know how to create a new `PMACC_CONST_VECTOR` where the entries are the results of the division of another `PMACC_CONST_VECTOR` by a constant `SI::ATOMIC_UNIT_ENERGY`

I'm well aware of that we don't want numbers in there ... it's just that I have no idea how to define that object generically